data/json/itemgroups/magazines.json

```diff
@@ -106,6 +109,8 @@
     "id" : "mags_pistol_rare",
     "//" : "Aftermarket or otherwise uncommon pistol magazines excluding milspec.",
     "items": [
+      ["glock_drum_50rd", 35],
```

Comment:
these drum magazines are going to be incredibly rare, drop them to like... 5 or so.

Reply:
I dropped them 10%.

Reply:
5

Reply:
Okay.

Reply:
Done!